src/tutorial/adding-a-skelton-of-denops-plugin.md

```diff
@@ -0,0 +1,23 @@
+# Adding a Skelton of Denops Plugin
+
+Once a Denops plugin is loaded, Denops calls the `main` function exported from `main.ts` of the plugin code.
+So initially you can write `main.ts` like:
+
+```ts:main.ts
+import { Denops } from "https://deno.land/x/denops_std@v1.0.0-beta.2/mod.ts";
+
+export async function main(denops: Denops): Promise<void> {
+  // Plugin program starts from here
+  console.log("Hello Denops!");
+};
+```
+
+An argument `denops` is passed to the `main` function; where `denops` is an instance of `Denops` class exported from [denops-std][].
+
+Then you restart vim, and you can see a message `[denops] Hello Denops!` on the vim window.
+
+![](https://storage.googleapis.com/zenn-user-upload/y8fyu6tap3tapjbxtltni6jynq8y)
```

Comment:
**must**

Please do NOT refer to images on Zenn directly. Instead, create an issue with "Resources" or whatever on this project and copy and paste images from Zenn to that. Then refer to newly copied images on this project issue.

Reply:
Please use https://github.com/vim-denops/denops-documentation/issues/6 issue to paste screenshots.

Reply:
Saved images under `src/img/...` at 2138405.